### ISSUE TITLE
Feature request: silent injection.

### ISSUE THREAD

**Issue by gir489returns** [nefarius/Injector#21]
I don't need the console to open and then close when using a batch file. It's just annoying to see it flash open then close itself. You should allow for the user to pass a parameter that silently injects. If an error occurs, maybe display it with a message box.

**Comment by nefarius** [comment]
That's the whole point behind a **command line utility**; to run it from a terminal. Ofc. it will pop up when used in batch, that is by design. If you want no window, change the way you call it, e.g. from C# you can suppress window creation.

**Comment by andry81** [comment]
> That's the whole point behind a **command line utility**; to run it from a terminal. Ofc. it will pop up when used in batch, that is by design. If you want no window, change the way you call it, e.g. from C# you can suppress window creation.

This is not related to "command line utility". Windows opens the console window for the executable because it has been linked with the Console application flag: `/SUBSYSTEM:CONSOLE`

https://learn.microsoft.com/en-us/cpp/build/reference/subsystem-specify-subsystem

You can link "Injector-gui.exe" with the GUI application flag: `/SUBSYSTEM:WINDOWS`

**Comment by nefarius** [comment]
Let me rephrase then: it is a console tool on purpose. The end.
